src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -614,7 +622,7 @@ pub struct HardcodedLayers {
 
     pub viz_fullscreen : Layer,
     pub breadcrumbs    : Layer,
-    layers             : Layers,
+    layers             : Layers,    
```

Comment:
You left spaces on the end of the line, which is strange, as IDEs should trim it on save